qiskit_experiments/framework/composite/parallel_experiment.py

```diff
@@ -13,13 +13,100 @@
 Parallel Experiment class.
 """
 
+import itertools
+from collections import defaultdict
+from typing import List
+
 from qiskit import QuantumCircuit, ClassicalRegister
 
 from .composite_experiment import CompositeExperiment
 
 
 class ParallelExperiment(CompositeExperiment):
-    """Parallel Experiment class"""
+    """Parallel Experiment class.
+
+    This experiment takes multiple experiment instances and generates
```

Comment:
This added docs should probably be moved into module level docs somewhere since they are very long for class docs